docs/examples/lisflood.ipynb

```diff
@@ -5,131 +5,178 @@
    "id": "c02a5bfe",
```

Comment:
<p>You are zooming in on some catchment Can you mention in the top cell which catchment we ran lisflood for?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>An explanation is added. </p>